code/modules/research/designs/tool_designs.dm

```diff
@@ -93,6 +93,16 @@
 	category = list("Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
 
+/datum/design/floorpainter
```

Comment:
Why do we have two of these

Reply:
one is for the autolathe and the other for the protolathe